### PR TITLE
[Asset] [Natsuki]: Plastic Wrap Roll

### PR DESCRIPTION
**Pull request by Codefreak64**
for a while now the plastic wrapping has been without its roll, this asset inclusion adds a new member to the Spankingtoys group, a roll of plasticwrap.

-adds spankingtoysPlasticWrap to the hand item slot
-wrap will be given to those who bought the wrap already (added into the buy group)
-gives more item inclusion to RP (never intended but good to have)

### CODE REVIEW

**Comment by Codefreak64** [comment]
one sec, let me look into what i did wrong

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 1]
```diff
@@ -3382,6 +3382,10 @@ var AssetFemale3DCG = [
 				Name: "SpankingToysScissors", Value: 15, Random: false, DynamicAllowInventoryAdd: () => { return false; },
 				PrerequisiteBuyGroups: ["SpankingToys"], PreviewIcons: ["Handheld"]
 			},
+			{
+				Name: "SpankingToysPlasticWrap", Value: 100, BuyGroup: "PlasticWrap", Random: false, DynamicAllowInventoryAdd: () => { return false; },
+        PrerequisiteBuyGroups: ["SpankingToys"], PreviewIcons: ["Handheld"]
+     		},
```
Comment (Jomshir98) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:3388]:
```suggestion
				PrerequisiteBuyGroups: ["SpankingToys"], PreviewIcons: ["Handheld"]
			},
```

File: BondageClub/Screens/Inventory/ItemHands/SpankingToys/SpankingToy.js [hunk 2]
```diff
@@ -186,6 +186,11 @@ const InventoryItemHandsSpankingToysOptions = [
 		Property: { Type: "Scissors" },
 		ExpressionTrigger: []
 	},{
+
+		Name: "PlasticWrap",
+		Property: { Type: "PlasticWrap" },
+		ExpressionTrigger: []
+  },{
```
Comment (Jomshir98) [inline BondageClub/Screens/Inventory/ItemHands/SpankingToys/SpankingToy.js:193]:
```suggestion
	},{
```

**Comment by Codefreak64** [comment]
Thank you Jomshir98 for helping out with resolving, trying to get back into asset making now that freed up again. one step closer to joining the team I hope

**Comment by Ben987** [comment]
Cute!  I like it.  Thanks :)

**Pull request merged**